### ISSUE TITLE
Travis CI ignores local_dir when deploying to AWS Elastic Beanstalk

### ISSUE THREAD

**Issue by thathurtabit** [travis-ci/travis-ci#9716]
Hello all,

I'm new to Travis CI and to AWS / Elastic Beanstalk, but I may have run into an issue.

My `.travis.yml` file (after successfully building and deploying) seems to ignore the `local_dir: build` and deploys everything _other_ than my `/build` folder. When I check my S3 bucket, it has everything other than the `/build` folder - and website displays the unminifed version. (I'm using [create-react-app](https://github.com/facebook/create-react-app), fyi).

I've used this [Travis doc](https://docs.travis-ci.com/user/deployment/s3#Deploy-From-Only-One-Folder) for reference.

```
language: node_js
node_js:
  - stable
cache:
  directories:
  - node_modules
script:
  - npm test
  - npm run build
deploy:
  provider: elasticbeanstalk  
  region: [my-region]  
  app: [my-app]
  env: [my-env]
  access_key_id: [my_access_id]
  secret_access_key:
    secure: [my_access_key]
  bucket_name: [my-bucket-name]
  skip_cleanup: true
  local_dir: build
  on:
    branch: master
```

**Comment by thathurtabit** [comment]
This issue is related to: https://github.com/travis-ci/dpl/issues/172
TL;DR (use `zip_file:`) option.
